platform/ios/src/MGLMapView.mm

```diff
@@ -4605,7 +4606,7 @@ - (void)updateHeadingForDeviceOrientation
     return [self visibleFeaturesAtPoint:point inStyleLayersWithIdentifiers:nil];
 }
 
-- (NS_ARRAY_OF(id <MGLFeature>) *)visibleFeaturesAtPoint:(CGPoint)point inStyleLayersWithIdentifiers:(NS_SET_OF(NSString *) *)styleLayerIdentifiers
+- (NS_ARRAY_OF(id <MGLFeature>) *)visibleFeaturesAtPoint:(CGPoint)point inStyleLayersWithIdentifiers:(NS_SET_OF(NSString *) *)styleLayerIdentifiers withPredicate:(NSPredicate *)predicate
```

Comment:
Renaming `-visibleFeaturesAtPoint:inStyleLayersWithIdentifiers:` to `-visibleFeaturesAtPoint:inStyleLayersWithIdentifiers:withPredicate:` breaks backwards compatibility. Checking for nil below isn’t enough, because this signature now requires the developer to _specify_ nil rather than omitting that part of the selector. (In Swift, this is the difference between an optional argument and an argument with a default value.) You’ll need to bring back `-visibleFeaturesAtPoint:inStyleLayersWithIdentifiers:`, which can call into this method.